Rewrites/double quotes single

Quix0r opened the pull request:
This is more a clean-up than a rewrite:
- converted some double-quotes to single
- fixed some indenting
- added spaces around `=>` for readability

Quix0r committed:
Changed:
- removed discouraged ?>
- changed double-quotes to single

Quix0r committed:
Ops, we need this back! :-(

Quix0r committed:
Changes:
- changed more double-quotes to single
- `include` is not a function

Quix0r committed:
Changes:
- changed more double-quotes to single
- cleaned up js_upload/file-uploader/server/php.php a lot
- added some type-hints

Quix0r commented:
[js_upload/file-uploader/server/php.php](https://github.com/friendica/friendica-addons/pull/1265/files#diff-1032b6cd8350b1efdecae97a55563cd4732567c4a5fbd4cf1d3d9ac0718f055b) has been actually cleaned up.

Quix0r committed:
Changes:
- added some type-hints
- changed double-quotes to singl
- cleaned up file (e.g. wrong place for curly braces)

Quix0r committed:
leistungsschutzrecht_is_member_site() returns boolean value

Quix0r committed:
Changes:
- added more type-hints
- cleaned up some files (curly braces, spaces)

Quix0r committed:
Re-formatted a bit

Quix0r committed:
Ops, wrong type

Quix0r committed:
Wrong types, why call $b so often ...

Quix0r committed:
Ops

Quix0r committed:
Wrong type-hint + renamed $b -> $body

Quix0r committed:
Fixed wrong type-hint + "imported" class

Quix0r committed:
Wrong type-hint + renamed $b -> $body

Quix0r committed:
Wrong type-hint

Quix0r committed:
Changes:
- made variable $uriId camel-case

Quix0r committed:
Cleaned up

Quix0r commented:
I think I keep batching such changes to keep each changeset small.

MrPetovan commented:
80 files and 1,200 lines changed isn't a "small changeset" but I'll take it!

MrPetovan commented on calc/calc.php:
```diff
@@ -5,26 +5,28 @@
-
-
-function calc_init($a) {
+function calc_module()
+{
+}
```
Please keep this on a single line, it's a statement rather than an actual function definition. The simple existence of this method is checked to figure out if the addon offers a module.
```suggestion
function calc_module() {}
```

Quix0r replied:
I copied your text almost unmodified in:
````
/**
 * This is a statement rather than an actual function definition. The simple
 * existence of this method is checked to figure out if the addon offers a
 * module.
 */
````

MrPetovan commented on convert/convert.php:
```diff
@@ -188,24 +183,21 @@ function getTable($value, $from_unit, $to_unit, $precision) {
-
-
-		$o .= ($conv->getTable($_POST['value'], $_POST['from_unit'], $_POST['to_unit'], 5))."</p>";
+		$_POST['value'] = $_POST['value'] + 0;
+		$o .= ($conv->getTable($_POST['value'], $_POST['from_unit'], $_POST['to_unit'], 5)) . '</p>';
```
```suggestion
		$o .= ($conv->getTable(intval($_POST['value']), $_POST['from_unit'], $_POST['to_unit'], 5)) . '</p>';
```

Quix0r replied:
The used class `UnitConvertor` is last edited from 2002 and therefore contains a deprecated constructor + CRLF. I added some type-hints where I was sure about it and added `@TODO` to it to have the need to rewrite it to a composer-based package.

MrPetovan commented on impressum/impressum.php:
```diff
@@ -15,91 +16,107 @@
 }
 
-function impressum_module() {
+function impressum_module()
+{
 }
```
```suggestion
function impressum_module() {}
```

MrPetovan commented on impressum/impressum.php:
```diff
@@ -15,91 +16,107 @@
+	if (! $text == '') {
+		DI::page()['htmlhead'] .= '<link rel="stylesheet" type="text/css" href="'.DI::baseUrl()->get().'/addon/impressum/impressum.css" media="all" />';
+		$body .= '<div class="clear"></div>';
+		$body .= '<div id="impressum_footer">'.$text.'</div>';
```
```suggestion
	if ($text != '') {
		DI::page()['htmlhead'] .= '<link rel="stylesheet" type="text/css" href="' . DI::baseUrl()->get() . '/addon/impressum/impressum.css" media="all" />';
		$body .= '<div class="clear"></div>';
		$body .= '<div id="impressum_footer">' . $text . '</div>';
```

MrPetovan commented on namethingy/namethingy.php:
```diff
@@ -7,27 +7,33 @@
-function namethingy_module() {}
-
-function namethingy_content(&$a) {
+function namethingy_module()
+{
+}
```
```suggestion
function namethingy_module() {}
```

MrPetovan commented on tictac/tictac.php:
```diff
@@ -5,27 +5,27 @@
+function tictac_module()
+{
 	return;
 }
```
```suggestion
function tictac_module() {}
```

MrPetovan commented on webrtc/webrtc.php:
```diff
@@ -6,50 +6,54 @@
+	DI::config()->set('webrtc', 'webrtcurl', $url);
 }
 
 function webrtc_module() {
 	return;
 }
```
```suggestion
function webrtc_module() {}
```

Quix0r committed:
Changes:
- if a <addon>_module() function is empty, it is more a statement and not a
  real function

Quix0r commented:
Yes, I know. I should have more split it. Next time I do it.

Quix0r committed:
Changes:
- added type-hint + intval() (@MrPetovan)
- also changed `if (!$foo == '')` to `if ($foo != '')`
- added some spaces for better readability
